### DISCUSSION TITLE
Blue ouline when I click on popover

### DISCUSSION THREAD

**Discussion by Rusinas** [Akryum/v-tooltip#418] [Q&A] [answered]
How to disable blue outline when clicking on popover?

**Comment by flytomek** [comment, marked as answer]
Just set outline to 0 on focus in your SCSS:
```
.popover {
    &:focus {
        outline:0;
    }
}
```

**Comment by adriaanzon** [comment]
Removing the outline with CSS sacrifices keyboard accessibility.

The blue outline only appears when the `autoHide` prop is true (default), because it causes the tabindex attribute to be set to `0`. When you remove the tabindex using the browser's inspector, the blue outline disappears.

https://github.com/Akryum/v-tooltip/blob/782c86f81d529a705bf566825457cf97748ca0b8/src/components/Popover.vue#L24

It would be nice if there was a way to remove the tabindex while still having auto-hide enabled, so the popover still hides when clicking away.
